### ISSUE TITLE
Write proposal for sender-side routing

### ISSUE THREAD

**Issue by jimmyfagan** [CDCgov/prime-reportstream#9971]
### User Story
As a sender I want to be able to dictate where a message gets routed, so that the delivery of my message isn't dependent on receiver settings.

### Description/Use Case
This may be useful for ETOR (TI) and for ELIMS work.

In ELIMS, an ELIMS sender may want to send a message to a given state and want to do that regardless of that state receiver's given filters. In the ELIMS case, this would be one sender sending messages to potentially many different receivers.

For ETOR (TI), a sender may know exactly what receiver (like Trusted Intermediary) should receive a message, and may only ever send to that receiver. Having the sender indicate that receiver may be simpler than relying on existing filtering to enforce that 1:1 connection of sender to receiver.

These are slightly different use cases, but the proposal resulting from this story should address both.

### Risks/Impacts/Considerations
The implications on the receiver side should be considered when writing and reviewing the proposal. A receiver should have some say in who gets to send to them, and filters probably should not be bypassed altogether. Maybe a receiver can specify a list of trusted senders? Maybe sender-side routing replaces a jurisdictional filter check, but all other receiver filters are still applied? Whatever the solution, it should not introduce the possibility of receivers receiving undesired data without having the ability to prevent it.

### Dev Notes
It may be useful to reference the routeTo URL parameter from the legacy pipeline. This is a version of "sender-side routing", even if it might be preferable to have something controlled at the settings level.

It may be useful to reference [#8497](https://github.com/CDCgov/prime-reportstream/issues/8497) for some considerations and ideas on this topic.

### Acceptance Criteria
- [ ] A proposal has been written, reviewed, and checked into the repo. This proposal should include a plan for sender-side routing for both ELIMS and ETOR applications (in other words, for 1-to-1 sender-to-receiver relationships, and for 1-to-many sender-to-receiver relationships)
- [ ] Follow up tickets have been written and assigned to the appropriate epic(s)

**Comment by bishoyayoub** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-6182b02547c1130010f459db/issues/CDCgov/prime-reportstream/9971?planning-poker) with Zenhub @arnejduranovic @jack-h-wang @JessicaWNava @jimmyfagan @luis-pabon-tf @mkalish @thetaurean

**Comment by bishoyayoub** [comment]
@jimmyfagan @arnejduranovic Do we still need this ticket given that we are not doing sender side routing anymore?

**Comment by arnejduranovic** [comment]
@bishoyayoub yes but let's move it to icebox because we may need something like this in the future, and unassign any epics
